py/selenium/webdriver/remote/webelement.py

```diff
@@ -415,7 +416,7 @@ def find_element(self, by=By.ID, value=None) -> WebElement:
 
         return self._execute(Command.FIND_CHILD_ELEMENT, {"using": by, "value": value})["value"]
 
-    def find_elements(self, by=By.ID, value=None) -> list[WebElement]:
+    def find_elements(self, by=By.ID, value=None) -> List[WebElement]:
```

Comment:
What version of Python supports lower case `list`?

Reply:
Looks like 3.9 and we still need to support 3.8

Reply:
this pr is created from my previous branch which also created another [pr](https://github.com/SeleniumHQ/selenium/pull/11735) long time ago, but looks like that pr is somehow reverted. That's why this change is here.

Reply:
Ah, yeah, looks like it got removed when we updated syntax to 3.8